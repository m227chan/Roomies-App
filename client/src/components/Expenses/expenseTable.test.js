import React from "react";
import renderer from "react-test-renderer";
import ExpenseTable from "./expenseTable";

describe("ExpenseTable", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<ExpenseTable />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
