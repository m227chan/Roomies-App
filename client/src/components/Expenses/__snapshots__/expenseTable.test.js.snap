// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExpenseTable renders correctly 1`] = `
<div
  className="MuiPaper-root MuiTableContainer-root MuiPaper-elevation1 MuiPaper-rounded"
>
  <table
    aria-label="simple table"
    className="MuiTable-root"
    role={null}
  >
    <thead
      className="MuiTableHead-root"
      role={null}
    >
      <tr
        className="MuiTableRow-root MuiTableRow-head"
        role={null}
      >
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-head MuiTableCell-alignRight"
          scope="col"
        >
          Payer
        </th>
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-head MuiTableCell-alignRight"
          scope="col"
        >
          Payee
        </th>
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-head MuiTableCell-alignRight"
          scope="col"
        >
          Amount
        </th>
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-head MuiTableCell-alignRight"
          scope="col"
        >
          Tag
        </th>
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-head MuiTableCell-alignRight"
          scope="col"
        >
          Comments
        </th>
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-head MuiTableCell-alignRight"
          scope="col"
        >
          Date
        </th>
      </tr>
    </thead>
    <tbody
      className="MuiTableBody-root"
      role={null}
    />
  </table>
</div>
`;
